cmds/core/cmp/cmp.go

```diff
@@ -56,12 +56,12 @@ func emit(rs io.ReadSeeker, c chan byte, offset int64) error {
 	}
 }
 
-func openFile(name string) (*os.File, error) {
+func openFile(stdin *os.File, name string) (*os.File, error) {
```

Comment:
The name openFile is a bit odd given that it is passed a file?

not sure if there is a better name? readFileOrStdin? not sure.

Reply:
yep sounds good